cmd/asset-syncer/postgresql_utils.go

```diff
@@ -247,3 +247,14 @@ func (m *postgresAssetManager) insertFiles(chartFilesID string, files models.Cha
 	}
 	return err
 }
+
+// InvalidateCache for postgresql deletes and re-writes the schema
+func (m *postgresAssetManager) InvalidateCache() error {
+	tables := strings.Join([]string{dbutils.RepositoryTable, dbutils.ChartTable, dbutils.ChartFilesTable}, ",")
+	_, err := m.DB.Exec(fmt.Sprintf("DROP TABLE IF EXISTS %s CASCADE", tables))
```

Comment:
is the `CASCADE` necessary?

Reply:
Not here it's not, thanks. I'd originally only deleted the one table and relied on the cascades, but I had to change that as the files table doesn't have a FK in this PR. I've removed the CASCADE and will re-add it when the FK's are in the next PR (I think)>